CASMMON-446  Create victoriametrics documentation (#5423)
Cray-HPE/docs-csm: operations/system_management_health/Access_System_Management_Health_Services.md

logging in. The details of the security warning will indicate that a self-signed certificate/unknown issuer is being used for the site. Support for incorporation of certificates from Trusted Certificate
Authorities is planned for a future release.

### VictoriaMetrics

URL: `https://vmselect.cmn.SYSTEM_DOMAIN_NAME/select/0/prometheus/vmui`

Vmagent instance scrapes metrics from Kubernetes, Ceph, and the hosts (part of `victoria-metrics-k8s-stack` Helm chart).

> Victoria metrics generates alerts based on metrics and reports them to the Alertmanager. The 'Alerts' link at the top of the page will show all of the inactive, pending, and firing alerts on the system.

Reviewer: ```suggestion
VictoriaMetrics generates alerts based on metrics and reports them to the Alertmanager. The 'Alerts' link at the top of the page will show all of the inactive, pending, and firing alerts on the system.
```

Clicking on any of the alerts will expand them, enabling users to use the 'Labels' data to discern the details of the alert. The details will also show the state of the alert, how long it has been
active, and the value for the alert.

> For more information regarding the use of the victoria metrics interface, see

Reviewer: ```suggestion
For more information regarding the use of the VictoriaMetrics interface, see
```

> [Getting Started/](https://docs.victoriametrics.com/) in the victoria metrics online documentation.

Reviewer: ```suggestion
[Getting Started/](https://docs.victoriametrics.com/) in the VictoriaMetrics online documentation.
```
